src/patternfly/components/Scrollspy/scrollspy.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="pf-c-scrollspy{{#if scrollspy--modifier}} {{scrollspy--modifier}}{{/if}}"
```

Comment:
Should be `nav` in order for VO to announce it as such and probably should always have a label, even if it's hidden.